internal/ethapi/api.go

```diff
@@ -536,8 +536,7 @@ func (s *PublicBlockChainAPI) GetBalance(ctx context.Context, address common.Add
 	if state == nil || err != nil {
 		return nil, err
 	}
-	balance := state.GetOVMBalance(address)
-	return (*hexutil.Big)(balance), state.Error()
+	return (*hexutil.Big)(state.GetOVMBalance(address)), state.Error()
```

Comment:
I suppose one line of diff is better than 2 lines of diff but we start to have a lot going on on a single like when we combine things like this